tests/unit/source_code/linters/test_pyspark.py

```diff
@@ -98,25 +98,33 @@ def test_spark_sql_match_named(migration_index):
     "recoverPartitions",
     "refreshTable",
     "uncacheTable",
-    "table",
-    "insertInto",
-    "saveAsTable",
-]
+}
+TABLE_METHOD_NAMES = {
```

Comment:
Please also add some tests to prevent false positives, eg "foo.register()"

Reply:
Done in the functional tests.